src/mip/bound_strengthening.jl

```diff
@@ -0,0 +1,91 @@
+@doc raw"""
+
+The behavior of bound_strengthening!() depends on the ordering of the variables.
+
+Let the constraints in the j-th row be
+lrow ⩽ row_S x_S + row_j x_j ⩽ urow
+
+Denote the U(row_S x_S) be the upper bound of row_S x_S and
```

Comment:
You can use backticks and/or "math mode" typesetting with $'s in docstrings. E.g.
```suggestion
Denote by $U(row_S x_S)$ the upper bound of $row_S x_S$ and
```